addons/rating/models/mail_message.py

```diff
@@ -7,51 +7,42 @@
 class MailMessage(models.Model):
     _inherit = 'mail.message'
 
-    rating_ids = fields.One2many('rating.rating', 'message_id', groups='base.group_user', string='Related ratings')
+    rating_ids = fields.One2many("rating.rating", "message_id", string="Related ratings")
```

Comment:
Note: `groups` looks completely useless as .csv for rating is 1 only for internal users.